cmake/find-opensycl.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+from shutil import which
+import argparse
+import json
+
+
+def get_package_dir():
+  exe_name = which("syclcc")
```

Comment:
Maybe make the package dir a CMake variable? Just in case the `syclcc` in `PATH` is not the one that is wanted (or some environment variable is supposed to supersede this name), or not given at all.

Reply:
Hmmm. I would argue that the user is responsible for providing a correct `syclcc`. The executable is supposed to visible in the environment. Otherwise, nothing is going to get compiled.